src/components/views/ViewCampaign.jsx

```diff
@@ -518,6 +548,7 @@ ViewCampaign.propTypes = {
     }).isRequired,
   }).isRequired,
   balance: PropTypes.instanceOf(BigNumber),
+  isDelegate: PropTypes.func.isRequired,
```

Comment:
Don't pass isDelegate as a property... use new useContext hook.

Reply:
Or use consumer immediately above the component which needs whitelist provider values